src/tiledimage.js

```diff
@@ -1003,6 +1009,63 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         });
     },
 
+    /**
+     * TODO
+     */
+    setAjaxHeaders: function(ajaxHeaders, propagate){
```

Comment:
I think I'd prefer we don't have the special `null` API here, but instead, have two functions. `setAjaxHeaders` should always take headers in and it should be public. It can then call `updateAjaxHeaders` which would be private. The Viewer can call `updateAjaxHeaders`. 

Also, if you do pass null into here, it should clear out the local headers (replacing them with `{}` I suppose).

Reply:
Fair enough, I changed it in e51aa4a.